plugin-flink-source-file/src/main/java/io/github/interestinglab/waterdrop/flink/source/TextRowInputFormat.java

```diff
@@ -65,9 +65,7 @@ public void configure(Configuration parameters) {
 
     @Override
     public Row readRecord(Row reusable, byte[] bytes, int offset, int numBytes) throws IOException {
-        if (this.getDelimiter() != null && this.getDelimiter().length == 1
-                && this.getDelimiter()[0] == NEW_LINE && offset + numBytes >= 1
-                && bytes[offset + numBytes - 1] == CARRIAGE_RETURN) {
+        if (this.getDelimiter() != null && this.getDelimiter().length == 1 && this.getDelimiter()[0] == NEW_LINE && offset + numBytes >= 1 && bytes[offset + numBytes - 1] == CARRIAGE_RETURN) {
```

Comment:
TBH,I prefer the previous one, it looks clearer

Reply:
OK, I'll adjust it